src/pages/OpportunityDetails.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { MapPinIcon, CalendarDaysIcon, HeartIcon } from "@heroicons/react/24/outline";
+
+function OpportunityDetails() {
+  const opportunityData = {
+    title: "Green Guardians: Community Tree Planting Initiative",
+    location: "Los Angeles",
+    date: "August 30",
+    latitude: 34.0522,
+    longitude: -118.2437,
+    link: "https://google.com",
+    mainImageUrl: "../public/image1.webp",
+    galleryImages: [
+      "../public/image1.webp",
+      "../public/image1.webp",
+      "../public/image1.webp",
+      "../public/image1.webp",
+    ],
+    description: (
```

Comment:
Thinking forward - the description from the backend will be a simple string, so would mock it here as such. For now assumes it's just a single paragraph.

Reply:
Done!